code/__DEFINES/is_helpers.dm

```diff
@@ -89,19 +89,17 @@ GLOBAL_LIST_INIT(turfs_openspace, typecacheof(list(
 //More carbon mobs
 #define isalien(A) (istype(A, /mob/living/carbon/alien))
 
-#define isalienhumanoid(A) (istype(A, /mob/living/carbon/alien/humanoid))
-
 #define islarva(A) (istype(A, /mob/living/carbon/alien/larva))
 
-#define isalienadult(A) (istype(A, /mob/living/carbon/alien/humanoid) || istype(A, /mob/living/simple_animal/hostile/alien))
+#define isalienadult(A) (istype(A, /mob/living/carbon/alien/adult) || istype(A, /mob/living/simple_animal/hostile/alien))
```

Comment:
no you cannot regress the behavior of isalienadult existing and working for both carbons and simple aliens

Reply:
?